flink-connectors/flink-connector-gcp-pubsub/src/main/java/org/apache/flink/streaming/connectors/gcp/pubsub/source/README.md

```diff
@@ -0,0 +1,78 @@
+# Flink Source for Google Cloud Pub/Sub
+
+This is a source implementation for receiving Google Cloud Pub/Sub messages in Flink with an at-least-once guarantee.
+
+## Installation
+
+Add this dependency entry to your pom.xml to use the Google Cloud Pub/Sub source:
+
+```xml
+<dependency>
+  <groupId>org.apache.flink</groupId>
+  <artifactId>flink-connector-gcp-pubsub_2.11</artifactId>
+  <version>1.13-SNAPSHOT</version>
+</dependency>
+```
+
+## Usage
+
+Please keep in mind that the new source can be found in package `org.apache.flink.streaming.connectors.gcp.pubsub.source` while the old source implementation can still be found in `org.apache.flink.streaming.connectors.gcp.pubsub` (without `source` suffix). The API of the new source is not much different from the old one.
```

Comment:
I would not put this in here because eventually we want to remove the old connector when this is merged.